lua/nvim-tree/renderer.lua

```diff
@@ -381,6 +381,38 @@ function M.draw(tree, reload)
     update_draw_data(tree, show_arrows and 2 or 0, {})
   end
 
+  if view.is_help_ui() then
+    lines = {'HELP'}
+    hl = {{'NvimTreeRootFolder', 0, 0, string.len('HELP')}}
+    local bindings = view.View.bindings
+    local processed = {}
+    for i, v in pairs(bindings) do
+      if v:sub(1,35) == view.nvim_tree_callback('test'):sub(1,35) then
+        v = v:match("'[^']+'[^']*$")
+        v = v:match("'[^']+'")
+        table.insert(processed,{i,v,true})
+      else
+        v = '"' .. v .. '"'
+        table.insert(processed,{i,v,false})
+      end
+    end
+    table.sort(processed,function(a,b)
+      return (a[3]==b[3] and (a[2]<b[2] or (a[2]==b[2] and #a[1]<#b[1]))) or (a[3] and not b[3])
+    end)
+    local i, v, builtin
+    for num, val in pairs(processed) do
+      i = val[1]
+      v = val[2]
+      builtin = val[3]
+      local bind_string = string.format("%6s : %s",i,v)
+      table.insert(lines,bind_string)
+      local hl_len = math.max(6,#i)+2
+      table.insert(hl,{'NvimTreeFolderName', num, 0, hl_len})
+      if not builtin then
+        table.insert(hl,{'NvimTreeFileRenamed', num, hl_len, -1})
+      end
+    end
```

Comment:
could you factor this out in a function please ?

Reply:
also some formatting would be great

Reply:
Have factored the help drawing into `draw_help`.

I'm not sure what you mean by "formatting" here, could you clarify?

Reply:
looks good enough like this for now. I need to install some formatting tool and set it in the CI first